gunicorn/config.py

```diff
@@ -221,6 +221,17 @@ def paste_global_conf(self):
 
         return global_conf
 
+    @property
+    def server_tokens(self):
+        s = self.settings['server_tokens'].get()
+        if s == 'true':
+            return SERVER_SOFTWARE
+
+        if s == 'false':
```

Comment:
Might be nice to have the option to just simply disable the header altogether, not even leaking `gunicorn`.